src/main/java/Duke/Task/Deadline.java

```diff
@@ -0,0 +1,55 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline-type Task is a task consisting of its details and deadline in day.
+ */
+public class Deadline extends Task {
+	/**
+	 * The deadline in day.
+	 */
+	protected LocalDate by;
+	private static final DateTimeFormatter inputDateFormat = DateTimeFormatter.ofPattern("dd/MM/yyyy");
```

Comment:
I believe this is a constant, so it would be more appropriate to name `INPUT_DATE_FORMAT`? Same as the `outputDateFormat` below.